oss-internship-2020/gdal/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.10) 
+
+project(test CXX C)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+set(SAPI_ROOT "/usr/local/google/home/inach/work/sandboxed-api" CACHE PATH "Path to the Sandboxed API source tree")
```

Comment:
Do not use an absolute path here. If you want to provide a reasonable default, use this
```
set(SAPI_ROOT "${PROJECT_SOURCE_DIR}/../../.." CACHE PATH "Path to ...
```